core-bundle/contao/templates/modules/mod_two_factor.html5

```diff
@@ -35,6 +35,7 @@ $this->wrapperAttributes = $this
         </div>
         <div class="submit_container">
           <button type="submit" class="submit"><?= $this->trans('MSC.enable') ?></button>
+          <a href="<?= $this->targetPath ?>" class="tl_submit" ><?= $this->trans('MSC.cancelBT') ?></a>
```

Comment:
```suggestion
          <a href="<?= $this->targetPath ?>" class="submit" ><?= $this->trans('MSC.cancelBT') ?></a>
```

Reply:
Fixed in [709617b](https://github.com/contao/contao/pull/8007/commits/709617bd77d2d0fb4d48003fe311a3c132c33f8e).